mne/viz/topomap.py

```diff
@@ -1067,3 +1068,234 @@ def plot_evoked_topomap(evoked, times=None, ch_type='mag', layout=None,
         plt.show()
 
     return fig
+
+
+def _plot_topomap_multi_cbar(data, pos, ax, title=None, unit=None,
+                             vmin=None, vmax=None, cmap='RdBu_r',
+                             colorbar=False):
+    """Aux Function"""
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+    ax.set_yticks([])
+    ax.set_xticks([])
+    ax.set_frame_on(False)
+    vmin = np.min(data) if vmin is None else vmin
+    vmax = np.max(data) if vmax is None else vmax
+
+    if title is not None:
+        ax.set_title(title, fontsize=10)
+    im, _ = plot_topomap(data, pos, vmin=vmin, vmax=vmax, axis=ax,
+                         cmap=cmap, image_interp='bilinear', contours=False)
+
+    if colorbar is True:
+        divider = make_axes_locatable(ax)
+        cax = divider.append_axes("right", size="10%", pad=0.25)
+        cbar = plt.colorbar(im, cax=cax, format='%3.3f')
+        cbar.set_ticks((vmin, vmax))
+        if unit is not None:
+            cbar.ax.set_title(unit, fontsize=8)
+        cbar.ax.tick_params(labelsize=8)
+
+
+def plot_epochs_psd_topomap(epochs, bands=None, vmin=None, vmax=None,
+                            proj=False, n_fft=256, ch_type=None,
+                            n_overlap=0, layout=None,
+                            cmap='RdBu_r', agg_fun=np.mean, dB=False, n_jobs=1,
```

Comment:
shouldn't this `agg_fun` also be `None` by default like the other? in any case, it should be consistent